File: JavaScript/7/7_1.js
function calculateAverage (numbers){
  let total = 0;
    for(let i=0;i<numbers.length;i++){
  total+=numbers[i];
  }
  let average = total/numbers.length;
  return average;
}
  
  let scores = [80,90,100];
  let avgScore = calculateAverage(scores);
  console.log("Average Score: "+avgScore);
  

File: C#/7/7_1.cs
double CalculateAverage (double[] numbers)
{
    double total=0;

    for(int i=0;i<numbers.Length;i++)
    {
        total+=numbers[i];
    }

    double average=total/numbers.Length;

    return average;
}


double[] scores={80,90,100};
double avgScore=CalculateAverage(scores);

Console.WriteLine("Average Score: "+avgScore);
